src/app/app-initializer.ts

```diff
@@ -10,7 +10,8 @@ export function initApplication(store: Store<fromApp.State>): VoidFunction {
     store.dispatch(new fromApp.LoadFeatureToggleConfig([AppConstants.FEATURE_NAMES.feeAccount,
       AppConstants.FEATURE_NAMES.editUserPermissions,
       AppConstants.FEATURE_NAMES.caaMenuItems,
-      AppConstants.FEATURE_NAMES.newRegisterOrg]));
+      AppConstants.FEATURE_NAMES.newRegisterOrg,
+      AppConstants.FEATURE_NAMES.odgInviteUserFlow]));
```

Comment:
Should be 'ogd', not 'odg'. I believe this affects a few of the changes but will be corrected with a refactor.

Reply:
Good eye! Thanks 👍